mos-platform/rp6502/clock_gettimespec.c

```diff
@@ -0,0 +1,13 @@
+#include "rp6502.h"
+#include <time.h>
+
+int __clock_gettimespec(struct timespec *ts, unsigned char op)
+/* Internal method shared by clock_getres and clock_gettime. */
+{
```

Comment:
Nit: move the comment before the signature and place the brace on the opening line.